doc/subsystems/bluetooth/hci.txt

```diff
@@ -0,0 +1,994 @@
+of the event parameters. The subevent code shall be set to one of the valid
+subevent codes from a vendor specific event. All other subevent parameters are
+defined in the vendor specific events.
+
+
+Zephyr Fatal Error
+==================
+
+This event reports a fatal non-recoverable error from the local Controller.
+
++-------------------------------+------------+-------------------------------+
+| Event                         | Event Code | Event Parameters              |
++-------------------------------+------------+-------------------------------+
+| Fatal_Error                   | 0xFF       | Subevent_Code,                |
+|                               |            | Program_Counter,              |
+|                               |            | Error_Info                    |
++-------------------------------+------------+-------------------------------+
+
+The Error_Info provides a string representation of the occurred error. It may
+include information like file name and line numbers.
+
+	Subevent_Code:                                  Size: 1 Octet
+	+--------------------+--------------------------------------+
+	| Value              | Parameter Description                |
+	+--------------------+--------------------------------------+
+	| 0x02               | Subevent code for Fatal Error event  |
+	+--------------------+--------------------------------------+
+
+	Program_Counter:                               Size: 4 Octets
+	+--------------------+--------------------------------------+
+	| Value              | Parameter Description                |
+	+--------------------+--------------------------------------+
+	| 0xFFFFFFFF         | Program counter where the error      |
+	|                    | occurred                             |
+	+--------------------+--------------------------------------+
+
+	Error_Info:                                    Size: variable
+	+--------------------+--------------------------------------+
+	| Value              | Parameter Description                |
+	+--------------------+--------------------------------------+
+	|                    | UTF-8 encoded error information      |
+	+--------------------+--------------------------------------+
+
+The event is reported by default unless masked away by Set_Event_Mask command.
+
+
+Zephyr Trace Information Event
+==============================
+
+This event reports the link manager and link layer trace information.
+
++-------------------------------+------------+-------------------------------+
+| Event                         | Event Code | Event Parameters              |
++-------------------------------+------------+-------------------------------+
+| Trace_Information             | 0xFF       | Subevent_Code,                |
+|                               |            | Trace_Type,                   |
+|                               |            | Trace_Data                    |
```

Comment:
should there be length here - this is variable length, right?  (or in common header)